components/infobox/wikis/starcraft2/infobox_league_custom.lua

```diff
@@ -323,6 +360,7 @@ function CustomLeague:defineCustomPageVariables(args)
 	Variables.varDefine('headtohead', args.headtohead or 'true')
 	Variables.varDefine('tournament_maps', Json.stringify(args.maps))
 	Variables.varDefine('tournament_series_number', args.number and string.format('%05i', args.number) or nil)
+	Variables.varDefine('matchDate', self.data.startTime.storage)
```

Comment:
Can we change `matchDate` to `match_date`?

Reply:
it has to be the same var name as in match input processeing, which is `matchDate`

Reply:
Can we cahnge it there too :D?

Reply:
ehm that needs quite a few additional changes in several templates/modules (non git)
and that on sc2 and broodwar

Reply:
```suggestion
	Variables.varDefine('match_date', self.data.startTime.storage)
```

Reply:
now depends on #4783